src/test/java/com/exasol/adapter/jdbc/ColumnMetadataReaderTest.java

```diff
@@ -105,40 +104,37 @@ void testParseDouble(final int typeId) throws SQLException, RemoteMetadataReader
     @ParameterizedTest
     void testParseCharWithSize(final int typeId) throws SQLException, RemoteMetadataReaderException {
         final int size = 70;
-        assertSqlTypeWithPrecisionConvertedToExasolType(typeId, size, DataType.createChar(size, ExaCharset.UTF8));
+        assertSqlTypeWithPrecisionConvertedToExasolType(typeId, size, 0, DataType.createChar(size, ExaCharset.UTF8));
     }
 
-    private void assertSqlTypeWithPrecisionConvertedToExasolType(final int typeId, final int expectedPrecision,
+    private void assertSqlTypeWithPrecisionConvertedToExasolType(final int typeId, final int size, final int scale,
```

Comment:
```suggestion
    private void assertSqlTypeWithPrecisionConvertedToExasolDouble(final int typeId, final int size, final int scale,
```
I think this is more precise.

Reply:
It shouldn't be double. It's an original type name that is not used for mapping because we use a data type number. I'm not sure if we need this field at all.